docs/en/observability/view-monitor-status.asciidoc

```diff
@@ -82,7 +82,7 @@ an assigned status is derived from the threshold values in the <<configure-uptim
 [role="screenshot"]
 image::images/tls-certificates.png[TLS certificates]
 
-The table entries can be sorted by _status_ and _valid until_. You can use the search bar at the
+The table entries can be sorted by _status_ and _valid until_. Use the search bar at the
```

Comment:
I would keep the original as it explains to the user what they can do if required.